test/arbitrage.test.js

```diff
@@ -252,7 +486,15 @@ test('select dataframe', t => {
 
 
 // ADD MORE TEST CASES
-test.todo('select vector using predicate')
+test('select vector using predicate', t => {
```

Comment:
Need more tests here. There's an obvious test, which is what if the function is not a valid predicate?